week1/homework/src/server.js

```diff
@@ -10,6 +10,30 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    function sendResponse(status, stateValue, contentType) {
+      state = stateValue;
+      response.writeHead(status, { 'Content-Type': contentType });
+      response.write(`{ "state": ${state}}`);
```

Comment:
This will work but if your object gets more complicated, this way of doing things can get hairy. We have a nice util to translate an object into a JSON string: `JSON.stringify()`.